bolt/lib/Profile/YAMLProfileReader.cpp

```diff
@@ -456,6 +435,39 @@ Error YAMLProfileReader::readProfile(BinaryContext &BC) {
       ++MatchedWithLTOCommonName;
     }
   }
+  return MatchedWithLTOCommonName;
+}
+
+Error YAMLProfileReader::readProfile(BinaryContext &BC) {
+  if (opts::Verbosity >= 1) {
+    outs() << "BOLT-INFO: YAML profile with hash: ";
+    switch (YamlBP.Header.HashFunction) {
+    case HashFunction::StdHash:
+      outs() << "std::hash\n";
```

Comment:
BC.outs()

Reply:
I'm erring on the side of making minimal code change - although it's showing up on gh as code added, I haven't touched the prologue of readProfile. If you see the large "deleted" section above (starting on the original line 353 of YAMLProfileReader.cpp) it's the exact same. So, I'd like to keep this PR just about refactoring function matching.

Reply:
@ayermolo, we didn't switch Profile component to BC logger class. That would be a separate effort.